module/Photo/src/Photo/Service/Album.php

```diff
@@ -321,6 +336,25 @@ public function getCreateAlbumForm()
         }
         return $this->sm->get('photo_form_album_create');
     }
+    
+    public function getMemberAlbum($lidNr)
+    {
```

Comment:
Perhaps it would be better to use a query to do this. You could join the tags and the photos table and then order by the photo date. This might also make pagination easier?